docs/reference/tools-reference/llm-tool.md

```diff
@@ -65,10 +65,12 @@ Setup connections to provisioned resources in prompt flow.
 | stop                   | list        | the stopping sequence for the generated text. Default is null.                                 | No       |
 | top_p                  | float       | the probability of using the top choice from the generated tokens. Default is 1.               | No       |
 | presence\_penalty      | float       | value that controls the model's behavior with regards to repeating phrases. Default is 0.      | No       |
-| frequency\_penalty     | float       | value that controls the model's behavior with regards to generating rare phrases. Default is 0. | No       |
+| frequency\_penalty     | float       | value that controls the model's behavior with regards to generating rare phrases. Default is 0.| No       |
 | logit\_bias            | dictionary  | the logit bias for the language model. Default is empty dictionary.                            | No       |
-
-
+| user                   | string      | a unique identifier representing your end-user. Default is empty string.                       | No       |
+| function\_call         | object      | value that controls which function is called by the model. Default is null.                    | No       |
+| functions              | list        | a list of functions the model may generate JSON inputs for. Default is null.                   | No       |
+| response_format        | string      | an object specifying the format that the model must output. Default is null.                   | No       |
```

Comment:
Is type `object`? The doc change may need to wait before implementation is shipped.